growth/too/templates/plan_manual.html

```diff
@@ -58,10 +62,14 @@
                 </div>
             </div>
             <div class=form-row>
-                <div class="form-group col-sm-12">
+                <div class="form-group col-sm-6">
                     <label for=plan_name>Queue name</label>
                     {{ render_field(form.queue_name, placeholder='Queue name') }}
                 </div>
+                <div class="form-group col-sm-6">
+                    <label for=subprogram_name>Subprogram name</label>
```

Comment:
Would you mind adding the subprogram name choices underneath (like for the filters), i.e. GW, GRB, Neutrino, just to clarify what this field is supposed to be?